Makefile

```diff
@@ -9,7 +9,7 @@ bins: temporal-server temporal-cassandra-tool temporal-sql-tool tdbg
 all: clean proto bins check test
 
 # Used in CI
-ci-build-misc: print-go-version proto buf-breaking bins temporal-server-debug shell-check copyright-check go-generate goimports-all gomodtidy ensure-no-changes
+ci-build-misc: print-go-version proto go-generate buf-breaking bins temporal-server-debug shell-check copyright-check goimports-all gomodtidy ensure-no-changes
```

Comment:
Need to generate the mock for workflowserviceclient or the code can't compile. Some mock usages are not in *_test.go (those exported test utils like ResourceTest). 

But in general I feel go-generate could potentially generate code needed by non-test logic, so make sense to move it forward.

Reply:
I reverted this change and moved the mockgen logic for workflowserviceclient & operationserviceclient into protoc.sh since the entire api package is controlled by that script.